Moves all unit test files to single folder under '/tests'

dmallory42 opened the pull request:
Fixes #759 

#### Changes proposed in this Pull Request

* Moves all unit test files into a folder under `/test` called `/unit`.
* Updates to the `phpunit` config file to correctly find these tests.

#### Testing instructions

* Run the command `npm run test:php` from the project root and verify that the tests run and that all tests are successful.

-------------------

- [ ] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

luizreis reviewed (commented):
@dmallory42 - thanks for working on this! The approach looks good and the tests work locally and on CI.

What do you think of also moving the `bootstrap.php` file to the new `unit` folder? That file is only used to set up the unit tests and leaving it under `tests` might be a bit confusing.

dmallory42 commented:
Thanks for reviewing @luizreis, I've updated to move the `bootstrap.php` file into the `unit` folder and also added a variable to store the directory we want to load the plugin from.

luizreis reviewed (approved):
LGTM! Tests run 💯, locally and on CI. :shipit:
